.github/workflows/multi-arch_compliance_operator_build.yml

```diff
@@ -0,0 +1,80 @@
+name: Multiarch build
```

Comment:
It would be good to have a descriptive name for this that's different from the one below.

Reply:
Added